From 3267a42e06b3119184af3ed151db8281b0e20d06 Mon Sep 17 00:00:00 2001
From: Sandor Yu <R01008@freescale.com>
Date: Fri, 29 May 2015 16:59:45 +0800
Subject: [PATCH] MLK-10256-01: imx clk: Set mipi csi clock parent to sys_pfd3

Set mipi csi root clock parent to sys_pfd clock.

Signed-off-by: Sandor Yu <R01008@freescale.com>
---
 arch/arm/mach-imx/clk-imx7d.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/mach-imx/clk-imx7d.c b/arch/arm/mach-imx/clk-imx7d.c
index cca6148..9d9ec6e 100644
--- a/arch/arm/mach-imx/clk-imx7d.c
+++ b/arch/arm/mach-imx/clk-imx7d.c
@@ -918,6 +918,8 @@ static void __init imx7d_clocks_init(struct device_node *ccm_node)
 	/* set lcdif pixel root clock source to get the required 33Mhz clock */
 	imx_clk_set_parent(clks[IMX7D_LCDIF_PIXEL_ROOT_SRC], clks[IMX7D_PLL_VIDEO_POST_DIV]);
 
+	imx_clk_set_parent(clks[IMX7D_MIPI_CSI_ROOT_SRC], clks[IMX7D_PLL_SYS_PFD3_CLK]);
+
 	/* set parent of SIM1 root clock */
 	imx_clk_set_parent(clks[IMX7D_SIM1_ROOT_SRC], clks[IMX7D_PLL_SYS_MAIN_120M_CLK]);
 
-- 
1.8.0

